src/Umbraco.Cms.Api.Management/Controllers/User/UpdateUserController.cs

```diff
@@ -17,20 +22,37 @@ public class UpdateUserController : UserControllerBase
     private readonly IUserService _userService;
     private readonly IUserPresentationFactory _userPresentationFactory;
     private readonly IBackOfficeSecurityAccessor _backOfficeSecurityAccessor;
+    private readonly IAuthorizationService _authorizationService;
 
     public UpdateUserController(
         IUserService userService,
-        IUserPresentationFactory userPresentationFactory, IBackOfficeSecurityAccessor backOfficeSecurityAccessor)
+        IUserPresentationFactory userPresentationFactory,
+        IBackOfficeSecurityAccessor backOfficeSecurityAccessor,
+        IAuthorizationService _authorizationService)
```

Comment:
```suggestion
        IAuthorizationService authorizationService)
```

Bad parameter naming - causes the need for `this._authorizationService = _authorizationService;` below.